python/pyspark/sql/streaming/readwriter.py

```diff
@@ -89,8 +92,8 @@ def format(self, source: str) -> "DataStreamReader":
 
         Examples
         --------
-        >>> spark.readStream.format("text")
-        <pyspark.sql.streaming.readwriter.DataStreamReader object ...>
+        >>> spark.readStream.format("text") # doctest: +ELLIPSIS
```

Comment:
Is `# doctest: +ELLIPSIS` needed? I didn't add in my PR (#40586)

Reply:
Hmm I believe you added these? https://github.com/apache/spark/pull/40586/files#r1157045341

Reply:
I didn't add. E.g.:
<img width="758" alt="image" src="https://user-images.githubusercontent.com/502522/230995675-b6d23836-5d49-42f3-967d-7cf04229a5f8.png">

Reply:
Ah I see, in line 538 there is one. I could remove all of the `# doctest: +ELLIPSIS` as we already defined them in the option: https://github.com/apache/spark/blob/17720b7121eee050a546126fad7ce8229fe6bda2/python/pyspark/sql/streaming/readwriter.py#L1504

https://stackoverflow.com/questions/17092215/how-enable-ellipsis-when-calling-python-doctest

Reply:
Actually maybe it doesn't hurt to put them there..? We already have this flag set, e.g.:  https://github.com/apache/spark/blob/master/python/pyspark/sql/streaming/query.py#L222

Reply:
No need to add them. It makes it the test less readable.